server/Cargo.toml

```diff
@@ -75,7 +75,7 @@ rusqlite = { version = "^0.9.5", features = ["bundled"] }
 time = "^0.1"
 tokio-core = "^0.1"
 toml = "^0.1"
-trust-dns = { version = "^0.12", path = "../client" }
+trust-dns = { version = "^0.12", path = "../client", default-features = false }
```

Comment:
this is ugly, isn't it... Should we just make client have an empty set of default features?

Reply:
That's what I would recommend we do. I think we should do it in its own separate follow-up to this PR since it's a bigger backward compatibility issue for the -client crate.

Reply:
Agreed.

Reply:
I made dnssec-openssl a non-default feature of -client in PR #281.